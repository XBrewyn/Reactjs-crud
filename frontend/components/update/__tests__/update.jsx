import React from 'react';
import { create } from 'react-test-renderer';
import Update from '../update';

describe('<Update />', () => {
  it('Renders correctly', () => {
    const component = create(<Update />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
