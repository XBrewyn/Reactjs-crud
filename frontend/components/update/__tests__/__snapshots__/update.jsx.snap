// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Update /> Renders correctly 1`] = `
<div
  className="css-1ht9v15"
>
  <h1>
    Update
  </h1>
  <form>
    <input
      name="id"
      onChange={[Function]}
      placeholder="id"
      type="text"
      value=""
    />
    <input
      name="name"
      onChange={[Function]}
      placeholder="name"
      type="text"
      value=""
    />
    <input
      name="password"
      onChange={[Function]}
      placeholder="password"
      type="password"
      value=""
    />
    <input
      onClick={[Function]}
      type="button"
      value="update"
    />
  </form>
</div>
`;
